internal/build.go

```diff
@@ -30,6 +32,19 @@ type BuildCommandParams struct {
 func RunBuild(args []string, params *BuildCommandParams) error {
 	buf := bytes.NewBuffer(nil)
 
+	var metadataFile = ""
+	rules, err := util.RetrieveRules(args)
+	if err == nil {
```

Comment:
If there is an error we don't want to cause problems at `build` time since this is just metadata

Reply:
As long as we account for this in the CLI and dashboards, I think we're fine.